src/ocaml_lsp.ml

```diff
@@ -40,4 +40,4 @@ let of_initialize_result (t : LanguageClient.InitializeResult.t) =
 
 let interfaceSpecificLangId t = t.interfaceSpecificLangId
 
-let canHandleSwitchImplIntf t = t.handleSwitchImplIntf
+let can_handle_switch_impl_intf t = t.handleSwitchImplIntf
```

Comment:
Both of these should be snake_case

Reply:
For the field names, perahps we should keep them to match the identifiers from capabilities payload?

Reply:
I mean that `let interfaceSpecificLangId` should be snake_case if `let can_handle_switch_impl_intf` is. The field names can stay the same.

Reply:
OK. I added a `has` prefix to match the `can` prefix of the other function.